src/e2e-test/features/gcsdelete/GCSDelete_WithMacro.feature

```diff
@@ -0,0 +1,46 @@
+@GCSDelete_Action
+Feature: GCS Delete - Verification of GCS Delete plugin with macro arguments
+
+  @GCS_CSV_TEST
+  Scenario: Verify GCS Delete successfully deletes all objects in gcs bucket when bucket name is passed as macro
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "GCS Delete" from the plugins list as: "Actions"
+    When Navigate to the properties page of plugin: "GCS Delete"
+    Then Enter the GCS Delete property "projectId" as macro argument "gcsDeleteProjectId"
+    Then Enter the GCS Delete property "objectsToDelete" as macro argument "gcsDeleteObjectsToDelete"
+    Then Enter the GCS Delete property "serviceAccountFilePath" as macro argument "gcsServiceAccount"
+    Then Validate "GCS Delete" plugin properties
+    Then Close the GCS Delete properties
+    Then Save and Deploy Pipeline
+    Then Run the Pipeline in Runtime
+    Then Enter runtime argument value "projectId" for key "gcsDeleteProjectId"
+    Then Enter runtime argument value bucket name for GCS Delete property objects to delete key "gcsDeleteObjectsToDelete"
```

Comment:
Similarly here,
`Then Enter runtime argument value bucket name for key "gcsDeleteObjectsToDelete"`

Reply:
Change done.